aeron-driver/src/main/java/io/aeron/driver/DriverNameResolverCache.java

```diff
@@ -79,10 +83,13 @@ void addOrUpdateEntry(
             entry.timeOfLastActivityMs = nowMs;
             entry.deadlineMs = nowMs + timeoutMs;
 
-            if (port != entry.port || byteSubsetEquals(address, entry.address, addressLength))
+            if (port != entry.port || !byteSubsetEquals(address, entry.address, addressLength))
             {
                 entry.address = Arrays.copyOf(address, addressLength);
                 entry.port = port;
+
+                eventLog.record(new RuntimeException(
```

Comment:
Should we record and `AeronEvent` here to save on the stack trace and allocation?

Reply:
Yes